lesson14/main.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+const wrapper = document.getElementById("js-wrapper");
+const ul = document.getElementById("js-ul");
+const modalBtn = document.getElementById("js-modalBtn");
+modalBtn.classList.add("btn");
+const modal = document.getElementById("js-modal");
+const openModalBtn = document.getElementById("js-openModalBtn");
+const jsonURL = "https://myjson.dit.upm.es/api/bins/j16z";
+const removeModal = () => {
+   modal.remove();
+   wrapper.classList.remove("bg_gray");
+};
+
+const startLoading = () => {
+   const loadingIcon = document.createElement("img");
+   loadingIcon.id = "loadingIcon";
+   loadingIcon.src = "img/loading-circle.gif";
+   wrapper.appendChild(loadingIcon);
+};
+
+const endLoading = () => {
+   const loadingIcon = document.getElementById("loadingIcon");
+   loadingIcon.remove();
+}
+
+openModalBtn.addEventListener("click", function () {
+   modal.style.display = "block";
+   wrapper.classList.add("bg_gray");
+   openModalBtn.remove();
+});
+
+modalBtn.addEventListener("click", () => {
+   const inputNumber = document.getElementById("js-number");
+   if (inputNumber.value !== "") {
+      console.log(inputNumber.value);
+      init();
+   } else {
+      alert("数字が未入力です。");
+   }
+});
```

Comment:
IMO
If you are going to implement it as later, it is better to compare with truthy values as much as possible to make the if statement easier to read.
I think it would be easier to read without the `else`.
In Japanese, it looks like this
`もし空だったらアラートを表示して処理を終了させる`

```suggestion
modalBtn.addEventListener("click", () => {
  const inputNumber = document.getElementById("js-number");
  if (inputNumber.value === "") {
    alert("数字が未入力です。");
    return;
  }
  console.log(inputNumber.value);
  init();
});
```

Reply:
Thank you.
I changed to compare the value with truthly values.
I thought it was certainly easier to read this one.
6de6d03